cmd/cmd.go

```diff
@@ -1165,13 +1165,13 @@ func NewCLI() *cobra.Command {
 
 	createCmd := &cobra.Command{
 		Use:     "create MODEL",
-		Short:   "Create a model from a Modelfile",
+		Short:   "Create a model from a Modelfile or the current directory (same as \"FROM .\")",
```

Comment:
```suggestion
		Short:   "Create a model from files",
```

Thanks for fixing this @rick-github! We may be able to simplify this one since the flags document the options